pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -228,50 +239,54 @@ func (o *ReconcileClusterRoleBindingsOptions) ChangedClusterRoleBindings() ([]*a
 
 // ReplaceChangedRoleBindings will reconcile all the changed system role bindings back to the recommended bootstrap policy
 func (o *ReconcileClusterRoleBindingsOptions) ReplaceChangedRoleBindings(changedRoleBindings []*authorizationapi.ClusterRoleBinding) error {
-	for i := range changedRoleBindings {
-		roleBinding, err := o.RoleBindingClient.Get(changedRoleBindings[i].Name)
-		if err != nil && !kapierrors.IsNotFound(err) {
-			return err
+	errs := []error{}
+	for _, changedRoleBinding := range changedRoleBindings {
```

Comment:
This change makes me nervous.  If anyone ever lets `changedRoleBinding` escape, it won't do what they expect since its a slice of pointers.